src/commcare_cloud/ansible/roles/elasticsearch/tasks/rolling_restart.yml

```diff
@@ -6,26 +6,46 @@
   retries: 20
   delay: 3
   changed_when: result.stdout.find('"acknowledged":true') != -1
+  tags: action_stop
+
+- debug: msg="{{ result }}"
```

Comment:
Did you want to keep this debug output?

Reply:
https://github.com/dimagi/commcare-cloud/pull/1990/commits/80d21becf069ed42949ddaafbd14ee922cffd37f